### PR TITLE
User/le2tang/movo network scripts

### PR DESCRIPTION
**Pull request by le2tang**
Change literal directory of env.sh file from /home/movo to /home/robohub
Fix network setup scripts that run when the workspace is sourced to use the correct IP addresses and hostnames
Change instances of ssh to user username robohub

### CODE REVIEW

**Review by FelixKHWong — DISMISSED** [review]

**Review by FelixKHWong — DISMISSED** [review]
I'll approve but make sure to document the usage of the scripts since its a big change to the running of the robot

File: movo_robot/movo_upstart/src/movo_upstart/job.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ def __init__(self, name="ros", interface=None, user=None, workspace_setup=None,
         # Fall back on current user as the user to run ROS as.
         self.user = user or getpass.getuser()
 
+        print(f"DEBUG: serlf.user = {self.user}, from {user} or {getpass.getuser()}")
```
Comment (nightingale-project) [inline movo_robot/movo_upstart/src/movo_upstart/job.py]: did u mean to push this?

**Review by nightingale-project — CHANGES_REQUESTED** [review]
- [x] Add instructions for accessing logs, stopping and starting sv, and remote master configuration
- [x] Post log output showing no nodes crashing

**Comment by le2tang** [comment]
Added :server movo1 minpoll 0 maxpoll 5 maxdelay .05" to /etc/chrony/chrony.conf on movo1

**Comment by le2tang** [comment]
> Added :server movo1 minpoll 0 maxpoll 5 maxdelay .05" to /etc/chrony/chrony.conf on movo1

This seems to have fixed the ntp issue as shown in the log

File: movo_network/movo_network_config.bash [hunk 2]
```diff
@@ -14,23 +14,23 @@
 # A remote PC is ROS slave, the default interface is lo (change it to whatever physical interface is connected to movo if working with real hardware),
```
Comment (nightingale-project) [inline movo_network/movo_network_config.bash:14]: this bash script uses movo1 & movo2 aliases. these do not always exist. dont have to resolve in this pr just something to remember https://github.com/nightingale-project/kinova-movo/issues/21

File: movo_network/movo_network_config.bash [hunk 3]
```diff
@@ -14,23 +14,23 @@
 # A remote PC is ROS slave, the default interface is lo (change it to whatever physical interface is connected to movo if working with real hardware), 
 #    and the IP address is DHCP from MOVO1
 
-if [ "$HOSTNAME" = MOVO1 ]; then
-    export ROBOT_NETWORK=eth0
-    export ROS_IP=$(ip -4 address show $ROBOT_NETWORK | grep 'inet' | sed 's/.*inet \([0-9\.]\+\).*/\1/')
-    export ROS_MASTER_URI=http://MOVO2:11311/
-elif [ "$HOSTNAME" = MOVO2 ]; then
+if [ "$HOSTNAME" = movo1 ]; then
+    export ROBOT_NETWORK=enp0s25
```
Comment (nightingale-project) [inline movo_network/movo_network_config.bash:18]: @alexswerner should this be the br0 interface? the change u left on the movo was only for movo2 case

**Comment by nightingale-project** [comment]
fyi @le2tang i added a small commit to this pr. hope thats fine.

File: movo_network/movo_network_config.bash (hunk 2 above)
Comment (alexswerner) [reply movo_network/movo_network_config.bash:14]: I think this uses $HOSTNAME which should be available independant of network connectivity.

File: movo_network/movo_network_config.bash (hunk 3 above)
Comment (alexswerner) [reply movo_network/movo_network_config.bash:18]: On movo1 br0 does not exist, only on movo2. So enp... is correct for movo1.

File: movo_network/movo_network_config.bash (hunk 2 above)
Comment (nightingale-project) [reply movo_network/movo_network_config.bash:14]: yes, $HOSTNAME will exist but won't line 24 fail? " export ROS_MASTER_URI=http://movo2:11311/ "

File: dev_notes.txt [hunk 4]
```diff
@@ -3,6 +3,10 @@ Make sure you are connected to the robohub wifi
 movo1 can be reached by `ssh robohub@movo1`
 movo2 can be reached by `ssh robohub@movo2`
 
+# Connecting RVIZ from a remote computer
+Run 'ROS_MASTER_URI="http://movo2:11311/"' on the local computer running rviz
```
Comment (younesr1) [inline dev_notes.txt:7]: is this outdated?

File: dev_notes.txt [hunk 5]
```diff
@@ -43,7 +47,7 @@ On movo1, run <rosrun si_utils torso_jtas_test.py>
 # Moving the gimbal
 Assert that the robot is not charging
 Assert estop is not engaged
-On movo1, run <roslaunch movo_bringup movo1.launch>
+On movo1, run <roslaunchmovo_bringup movo1.launch>
```
Comment (younesr1) [inline dev_notes.txt]: ?

File: dev_notes.txt [hunk 6]
```diff
@@ -57,3 +61,11 @@ Ensure you've run all your source commands (ie. source devel/setup.bash). These
 export ROS_MASTER_URI=http://129.97.71.49:11311
 export ROS_IP=<your ip address in same subnet as movo1>
 
+# Using the ROS upstart service
```
Comment (younesr1) [inline dev_notes.txt]: didnt u remove the upstart service?

**Review by younesr1 — APPROVED** [review]

**Comment by younesr1** [comment]
Master is broken. launching the basic launch files (movo1.launch, movo2.launch) yields errors. the errors seems to be related to networking. Im merging in this PR to hopefully fix master.

**Review by nightingale-project — APPROVED** [review]

**Pull request merged**